addOns/ascanrulesAlpha/src/main/resources/org/zaproxy/zap/extension/ascanrulesAlpha/resources/Messages_fr_FR.properties

```diff
@@ -166,3 +166,9 @@ ascanalpha.traceaxdscanner.otherinfo=Selon le code de la r\u00e9ponse, la vision
 ascanalpha.traceaxdscanner.refs=https\://msdn.microsoft.com/en-us/library/bb386420.aspx\nhttps\://msdn.microsoft.com/en-us/library/wwh16c6c.aspx\nhttps\://www.dotnetperls.com/trace
 ascanalpha.traceaxdscanner.soln=Examinez si la visionneuse de trace est r\u00e9ellement n\u00e9cessaire \u00e0 la production, et si ce n\u2019est pas le cas, d\u00e9sactivez-la. Si elle est n\u00e9cessaire , assurez-vous que son acc\u00e8s requiert authentification et autorisation. 
 
+ascanalpha.cors.info.name=En-t\u00eate CORS
+ascanalpha.cors.info.desc=Cross-Origin Resource Sharing (CORS) est un m\u00e9canisme bas\u00e9 sur les en-t\u00eates HTTP qui permet \u00e0 un serveur d'indiquer une autre origine (domaine, sch\u00e9ma d'URI ou port) que la sienne \u00e0 partir de laquelle le navigateur autorisera le chargement de ressources. Cela permet d'assouplir la Same-Origin Policy (SOP).
+ascanalpha.cors.vuln.name=Mauvaise configuration CORS
+ascanalpha.cors.vuln.desc=Cette mauvaise configuration CORS peut permettre \u00e0 un attaquant de r\u00e9aliser des requ\u00eates AJAX vers le site vuln\u00e9rable depuis une page malveillante charg\u00e9e par l'agent utilisateur de la victime.\nPour r\u00e9aliser des requ\u00eates AJAX authentifi\u00e9es, le serveur doit sp\u00e9cifi\u00e9 l'en-t\u00eate "Access-Control-Allow-Credentials: true" et l'en-t\u00eate "Access-Control-Allow-Origin" doit avoir la valeur null ou le domaine de la page malveillante. M\u00eame si cette mauvaise configuration ne permet pas de r\u00e9aliser des requ\u00eates authentifi\u00e9es, du contenu sensible accessible sans authentification peut quand m\u00eame \u00eatre r\u00e9cup\u00e9r\u00e9 (ex: les applications Web intranet).\nUne page malveillante peut \u00eatre h\u00e9berg\u00e9e par un site malveillant mais aussi un site de confiance avec des failles (ex: XSS, support de HTTP sans TLS permettant l'injection de code via un MITM, etc).
+ascanalpha.cors.soln=Si une ressource Web contient des informations sensibles, l'origine devrait \u00eatre correctement sp\u00e9cifi\u00e9e dans l'en-t\u00eate Access-Control-Allow-Origin. Seuls les sites de confiance ayant besoin de la ressource devraient \u00eatre sp\u00e9cifi\u00e9s dans l'en-t\u00eate, avec le protocole le plus s\u00e9curis\u00e9 support\u00e9.
+ascanalpha.cors.refs=https://developer.mozilla.org/fr/docs/Web/HTTP/CORS\nhttps://portswigger.net/web-security/cors
```

Comment:
This should all be reverted, only the main Messages.properties should be modified. Translations are handled via CrowdIn.